treepoem/__init__.py

```diff
@@ -112,7 +112,9 @@ def _get_ghostscript_binary():
 
 
 def _encode(data):
-    return codecs.encode(data.encode('utf8'), 'hex_codec').decode('ascii')
+    if isinstance(data, str):
```

Comment:
actually since we support Python 2 this should really check for `six.text_type`, since binary data there will be in a `str`